docs/docs-beta/docs/guides/monitor-alert/alerting.md

```diff
@@ -1,12 +1,322 @@
 ---
-title: "Setting up alerts"
+title: Setting up alerts on Dagster+
 sidebar_position: 30
-sidebar_label: "Alerting"
+sidebar_label: "Alerting on Dagster+"
 ---
+[comment]: <> (This file is automatically generated by `docs_beta_snippets/guides/monitor-alert/alerting/generate.py`)
 
-Alerting if my pipeline didn't execute
-Tracking when a run or sensor fails
-Knowing when a pipeline never ran
-Knowing if a pipeline is running slow, or an asset is late
-Knowing if my Dagster instance is having issues
+Dagster+ allows you to configure alerts to automatically fire in response to a range of events. These alerts can be sent to a variety of different services, depending on your organization's needs.
 
+These alerts can be configured in the Dagster+ UI, or using the `dagster-cloud` CLI tool.
+
+<details>
+<summary>Prerequisites</summary>
+- **Organization**, **Admin**, or **Editor** permissions on Dagster+
+</details>
+
+## Configuring a notification service
+
+To start, you'll need to configure a service to send alerts. Dagster+ current supports sending alerts through email, Microsoft Teams, PagerDuty, and Slack.
+
+<Tabs groupId="notification_service">
+  <TabItem value='email' label='Email'>
+    No additional configuration is required to send emails from Dagster+.
+
+All alert emails will be sent by `alerts@dagster.cloud`. Alerts can be configured to be sent to any number of emails.
+  </TabItem>
+  <TabItem value='microsoft_teams' label='Microsoft Teams'>
+    Create an incoming webhook by following the [Microsoft Teams documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=newteams%2Cdotnet).
+
+This will provide you with a **webhook URL** which will be required when configuring alerts in the UI (after selecting "PagerDuty" as your Notification Service) or using the CLI (in the `notification_service` configuration).
```

Comment:
```suggestion
This will provide you with a **webhook URL** which will be required when configuring alerts in the UI (after selecting **PagerDuty** as the Notification Service) or using the CLI (in the `notification_service` configuration).
```

Sorry for the nit ⛏️